src/PresentationalComponents/Inventory/Inventory.js

```diff
@@ -391,6 +391,27 @@ const Inventory = ({
     },
   };
 
+  const getActionsConfig = () => {
```

Comment:
Nice! It maybe even better to extract this util function out of the component into some helper file.

Reply:
Ok, I'll have a look into it, but there are a few values in there that are dependent / specific to the Inventory component.  It might be tricky to generalize the function.

Reply:
Aha, in that case, feel free to proceed please. Soon, we will release edge parity work and it is better to get this commit together.